### PR TITLE
chore: add jonaslagoni as maintainer

### PR DESCRIPTION
**Pull request by jonaslagoni**
As discussed in https://github.com/asyncapi/spec-json-schemas/issues/513#issuecomment-2060839618 I want to help maintain the schema files from a tooling perspective 💪

### CODE REVIEW

File: CODEOWNERS
Comment (smoya) [inline CODEOWNERS:12]: What's the meaning of "Tooling helpers"? And whats gonna be the difference between you and the maintainers in https://github.com/asyncapi/spec-json-schemas/pull/522/files#diff-fcf14c4b7b34fe7a11916195871ae66a59be87a395f28db73e345ebdc828085bR10 ? 🤔

Comment (smoya) [reply CODEOWNERS:12]: Personally, I wouldn't make such a distinction. I like what @derberg mentioned in https://github.com/asyncapi/spec-json-schemas/issues/513#issuecomment-2034861927: 

> we should treat spec-json-schemas as a tool, and focus more to have JSON Schema experts here

I believe all maintainers (atm, those at line 9) should be aware and able to review any PR.

Comment (jonaslagoni) [reply CODEOWNERS:12]: Alright 👍

**Review by smoya — APPROVED** [review]
LGTM! 🚀🌔

**Comment by smoya** [comment]
After merging this, @derberg will need to provide permissions to @jonaslagoni on this repo.

**Comment by derberg** [comment]
can't you also add @Pakisan in the same PR?

**Comment by jonaslagoni** [comment]
He added it here: https://github.com/asyncapi/spec-json-schemas/pull/523

**Comment by derberg** [comment]
ok, added you both to repo as Maintainers
merge conflict incoming in 3, 2, 1 ...

/rtm

**Pull request merged**
